util/formula_templater/formula.go

```diff
@@ -72,7 +72,10 @@ func printFormula(product, version, configLocation string, out io.Writer) error
 	return t.Execute(out, productConfig)
 }
 
-const formulaTemplate = `class {{ .Name }} < Formula
+const formulaTemplate = `# Copyright (c) HashiCorp, Inc.
```

Comment:
I vote to move these template constants into separate files and include them via `go:embeds`